cmplibrary/src/main/java/com/sourcepoint/cmplibrary/exposed/gpp/GppOptionTernary.kt

```diff
@@ -1,6 +1,6 @@
-package com.sourcepoint.cmplibrary.gpp.dto
+package com.sourcepoint.cmplibrary.exposed.gpp
 
-internal enum class GppTernaryType(
```

Comment:
Yes, it will be a part of our public API, since the main goal of the task is to provide our users with the ability to set their own `GppConfig`, with proper `GppOptions`